cmd/tetra/yaml/yaml.go

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package yaml
+
+import (
+	"fmt"
+
+	"github.com/cilium/tetragon/pkg/btf"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func New() *cobra.Command {
+	yamlCmd := &cobra.Command{
+		Use:   "yaml",
```

Comment:
Hi Jiri, could we consider `tetra tracingpolicy generate $arg` or something similar?

Also @kkourt mentioned about generating tracing policies, so maybe wait a bit for more feedback on the UI and then we decide ;-)

Reply:
sure, I don't mind changing the command name, let's see @kkourt opinion on this,
also I'll add doc, np